### PR TITLE
Converted the class based style in demo to function/hooks

### PR DESCRIPTION
**Pull request by RobKohr**

### CODE REVIEW

**Comment by DominicTobias** [comment]
Thanks I will merge this in but after I add back the commented stuff as I use that to test things ;) I'm working on something else at the moment so it will take some time before I get around to it

File: demo/demo.js [hunk 1]
```diff
@@ -1,75 +1,61 @@
 /* eslint-disable jsx-a11y/media-has-caption, class-methods-use-this */
-import React, { PureComponent } from 'react';
+import React, { useState } from 'react';
 import ReactDOM from 'react-dom'; // eslint-disable-line
 import ReactCrop from '../lib/ReactCrop';
```
Comment (martijndebruijn) [inline demo/demo.js:4]: I had to make to following changes to be able to compile the code.
import ReactCrop from 'react-image-crop';
import 'react-image-crop/dist/ReactCrop.css';

File: demo/demo.js [hunk 2]
```diff
@@ -89,72 +75,42 @@ class App extends PureComponent {
...
+        />
+      )}
+      {croppedImageUrl && (
+        <img alt="Crop" style={{ maxWidth: '100%' }} src={croppedImageUrl} />
+      )}
+    </div>
+  );
 }
 
 ReactDOM.render(<App />, cropEditor);
```
Comment (martijndebruijn) [inline demo/demo.js:116]: CropEditor does not exist.
I used:
ReactDOM.render(<App />, document.getElementById("root"));

**Review by martijndebruijn — CHANGES_REQUESTED** [review]

**Comment by DominicTobias** [comment]
Thanks happy to merge but there seems to be a conflict there ^

**Pull request closed without merging**